src/ui/Base/ComponentOptionsPostProcessing.ts

```diff
@@ -0,0 +1,31 @@
+import { each } from 'underscore';
+import { IComponentOptionsOption } from './IComponentOptions';
+import { Utils } from '../../utils/Utils';
+import { Logger } from '../../misc/Logger';
+
+export class ComponentOptionsPostProcessing<T> {
```

Comment:
I would prefer a noun instead of a verb for the name. e.g. `ComponentOptionsPostProcessor`.